studio.py

```diff
@@ -16,14 +16,15 @@
 @click.command()
 @click.option("--record/--replay", default=False)
 @click.option("--certutil", required=True, help="Path to certutil.")
+@click.option("--deterministic", required=True, help="Use deterministic JS in the recording")
```

Comment:
Is there a reason not to run with determinism? We can be opinionated here, so I think we should just make this the default and not provide an option to disable it.